docs/appendices/release-notes/6.0.0.rst

```diff
@@ -151,6 +151,15 @@ Scalar and Aggregation Functions
 Performance and Resilience Improvements
 ---------------------------------------
 
+- Added a :ref:`statement_max_length` setting to limit the length of allowed SQL
+  statements. It defaults to 65536. Statements exceeding this limit are rejected
+  because large statements consume a high amount of memory. Large statements are
+  typically caused by inlining values or using a large number of ``VALUES``
+  clauses.
+  Instead, statements should be written using parameter symbols (``?``) and
+  inserting lots of values should be done using either bulk operations or a
+  ``INSERT INTO`` in combination with a ``SELECT FROM`` and ``UNNEST``.
```

Comment:
maybe link to: https://cratedb.com/docs/guide/performance/inserts/methods.html#unnest

Reply:
This should be considered a breaking change. I am also almost sure, that a limit of 64K will lead to problems with some users on/after upgrade.

Reply:
>  I am also almost sure, that a limit of 64K will lead to problems with some users on/after upgrade.

Do you have concrete data to back this up?

Reply:
We have lots of examples of user select queries in double digit KB size in repositories and single table schemas that are close to that limit. I don't think we have any telemetry for statement size (yet).